server.js

```diff
@@ -29,15 +31,25 @@ app.use(cookieSession({
   name: 'session',
   keys: ['key']
 }));
+app.use(cookieParser());
 
 // Separated Routes for each Resource
 // Note: Feel free to replace the example routes below with your own
+const homeApiRoute = require('./routes/home');
+const authenticate = require("./routes/authenticate");
 const userApiRoutes = require('./routes/users-api');
+const categoriesApiRoutes = require('./routes/categories-api');
+const itemsApiRoutes = require('./routes/items-api');
 
 // Mount all resource routes
 // Note: Feel free to replace the example routes below with your own
 // Note: Endpoints that return data (eg. JSON) usually start with `/api`
+app.use('/home', homeApiRoute);
+app.use('/login', authenticate);
 app.use('/api/users', userApiRoutes);
+app.use('/api/categories', categoriesApiRoutes);
+app.use('/api/categories/:id', categoriesApiRoutes);
```

Comment:
//

Reply:
Not sure what you mean?

Reply:
I was curious whether this is needed since we already have `app.use('/api/categories',`

Reply:
👍No you're right, doesn't need it